frontend/src/citizen-frontend/calendar/AbsenceModal.tsx

```diff
@@ -230,14 +185,6 @@ export default React.memo(function AbsenceModal({
               </LineContainer>
               <CalendarModalSection>
                 <H2>{i18n.calendar.absenceModal.absenceType}</H2>
-                {contractDayAbsenceTypeSettings.visible &&
-                  !contractDayAbsenceTypeSettings.enabled && (
-                    <AlertBox
-                      message={
-                        i18n.calendar.absenceModal.contractDayAbsenceTypeWarning
```

Comment:
Localized text `contractDayAbsenceTypeWarning` could be removed from fi/sv/en.tsx.